### ISSUE TITLE
Docs update suggestion for installation on Ubuntu 22.04 and lower

### ISSUE THREAD

**Issue by e-buzyurkin** [bpftrace/bpftrace#3345]
I'm not sure that this is the correct way to suggest docs changes there in issues, but 

bpftrace version from apt on Ubuntu 22.04 jammy is [(0.14.0)](https://launchpad.net/ubuntu/jammy/+source/bpftrace). For my project I needed at least 0.17.x (strcontains function), so I've decided to build it from sources. 

I've followed instructions from INSTALL.md, but faced a problem with libbpf: manual suggests installing it from apt, as in the [Dockerfile.ubuntu](https://github.com/bpftrace/bpftrace/blob/master/docker/Dockerfile.ubuntu), but for Ubuntu 22.04 version is [0.5.0](https://launchpad.net/ubuntu/jammy/+source/libbpf), when 1.0 is required for bpftrace. 

My suggestion is to add instructions for installing libbpf from sources for Ubuntu 22.04 and lower. For Ubuntu 24.10+ apt installes latest 0.21.x, so the problem doesn't occur.

**Comment by ajor** [comment]
Good point! The instructions for building bpftrace from source are horribly out of date: https://github.com/bpftrace/bpftrace/blob/master/INSTALL.md#building-bpftrace

We should specify the minimum libbpf version requirement (commit dd589c3b31c13164bdc61ed174fbae6fe76c8308 or 1.5.0 when that comes out) and instructions on building it from source as you say.

I think we can also remove references to Ubuntu 18.04+18.10 and using at least Fedora 28 in order to get "newer kernels" :D

**Comment by historicattle** [comment]
Hello @ajor landed here from the good first issue. Could i contribute to this?

**Comment by viktormalik** [comment]
@historicattle sure! I just assigned the issue to you.

**Comment by Yokiatch** [comment]
Hello @viktormalik if the docs are still not updated can i work on them? Happy to hear your response

**Comment by jordalgo** [comment]
@Yokiatch Please feel free. Any PRs for improvements are most welcome!
